src/docfx/lib/github/GitHubAccessor.cs

```diff
@@ -231,7 +232,7 @@ HttpRequestMessage CreateHttpRequest(HttpContent content)
                 var message = new HttpRequestMessage();
                 message.Headers.Add("User-Agent", "DocFX");
                 if (!string.IsNullOrEmpty(_token))
-                    message.Headers.Add("Authorization", $"bearer {_token}");
```

Comment:
from here: https://developer.github.com/v4/guides/forming-calls/#communicating-with-graphql

It should be a bearer token